File: mobile/src/main/java/com/windscribe/mobile/about/AboutPresenter.java
/*
 * Copyright (c) 2021 Windscribe Limited.
 */

package com.windscribe.mobile.about;

import android.content.Context;

public interface AboutPresenter {

    void init();

    boolean isHapticFeedbackEnabled();

    void onAboutClick();

    void onBlogClick();

    void onJobsClick();

    void onPrivacyClick();

    void onStatusClick();

    void onTermsClick();

    void onViewLicenceClick();
}

File: mobile/src/main/java/com/windscribe/mobile/about/AboutPresenter.kt
/*
 * Copyright (c) 2021 Windscribe Limited.
 */
package com.windscribe.mobile.about

interface AboutPresenter {
    fun init()
    val isHapticFeedbackEnabled: Boolean
    fun onAboutClick()
    fun onBlogClick()
    fun onJobsClick()
    fun onPrivacyClick()
    fun onStatusClick()
    fun onTermsClick()
    fun onViewLicenceClick()
}